quires.sql

```diff
@@ -51,8 +51,7 @@ WHERE date_of_birth BETWEEN DATE '1990-01-01' AND DATE '2000-12-31'
 GROUP BY species;
 
 
-/* Vet clinic database: query multiple tables */
-
+/* Vet-clinic database: query multiple tables */
```

Comment:
@bobb-Rob 
thanks for your good and valuable feedback,
I have considered your all comments. and I tried to use the code you provided but the code you wroth on the comment is not working you can see the error below.
 
vet_clinic=# CREATE TABLE owners (
vet_clinic(# id INT NOT NULL IDENTITY(1,1),
vet_clinic(# full_name varchar(255),
vet_clinic(# age INT);
ERROR:  syntax error at or near "IDENTITY"
LINE 2: id INT NOT NULL IDENTITY(1,1),
                        ^
vet_clinic=# 

I couldn't find a solution for it could you help me but if want to **autoincremented PRIMARY KEY** the previouws code which I worth do that. 
![Creat-onwers-Table-error](https://user-images.githubusercontent.com/108926875/230610462-98ea0a57-4df6-46e4-ad00-1c638f9eea7b.JPG)
![Creat-onwers-Table-error](https://user-images.githubusercontent.com/108926875/230610674-ba2cf267-8792-44d6-83fd-12cfa1b282c8.JPG)


I don't know how to handle this issue I kindly request your to please help me according to this issue.